src/navigation/StackNavigator.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {createStackNavigator} from '@react-navigation/stack';
+import BookScreen from '../screens/Book/BookScreen.tsx';
+import PlayerScreen from '../screens/Player/PlayerScreen.tsx';
+import TabNavigator from './TabNavigator.tsx';
+import {NavigationContainer} from '@react-navigation/native';
+
```

Comment:
```suggestion

const Stack = createNativeStackNavigator<RootStackParamList>();

```

Вкажи тип коли створюєш стек.

Reply:
@G-Ro-Man  Зробив. Так само додав і до `createBottomTabNavigator<RootStackParamList>();`, але у мене єдиний тип RootStackParamList, який я передаю і в `createNativeStackNavigator` і в `createBottomTabNavigator`, це ок?

Reply:
ні, в тебе для табів має бути свої пропси з нестед пропсами, щось типу такого 

```
type TabParamList = {
  TabOne: NavigatorScreenParams<RootStackParamList>;
  TabTwo: { userId: string };
};
```

Reply:
Дякую! Виправив